ci-operator/step-registry/rosa/operator/install/rosa-operator-install-commands.sh

```
@@ -40,40 +40,60 @@ log "  PKO image: ${OPERATOR_PKO_IMAGE}"
+MIRROR_NS="ci-e2e-images"
+REGISTRY_ROUTE=$(oc get route default-route -n openshift-image-registry -o jsonpath='{.spec.host}' 2>/dev/null || true)
+
+if [[ -z "${REGISTRY_ROUTE}" ]]; then
+    log "WARNING: Internal registry route not found, falling back to direct pull"
 else
-    log "WARNING: Could not get CI registry credentials, PKO may fail to pull images"
+    log "Mirroring CI images to internal registry"
+    log "  Registry: ${REGISTRY_ROUTE}"
+
+    MIRROR_TMPDIR=$(mktemp -d)
+    trap "rm -rf ${MIRROR_TMPDIR}" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)